$MirOS: ports/www/dillo/v2/patches/patch-src_dns_c,v 1.1 2009/08/24 16:54:24 tg Exp $
--- src/dns.c.orig	Wed Feb  3 11:30:23 2010
+++ src/dns.c	Sun Feb 21 18:17:01 2010
@@ -34,6 +34,7 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
+#include <unistd.h>
 
 #include "msg.h"
 #include "dns.h"
